File: packages/core/src/config/audio.ts
// オーディオ設定（optional）。merged から `audio` を取り出し camelCase へ transform。

import { z } from "zod";

import { snakeToCamel } from "../../internal/case.ts";

/** `audio` セクション（optional）。 */
export const Audio = z
  .object({
    audio: z
      .object({
        chapter_max: z.number().default(100),
        target_duration_max: z.number().nullable().default(null),
        target_duration_min: z.number().nullable().default(null),
      })
      .strict()
      .prefault({}),
  })
  .transform((o) => snakeToCamel(o.audio));

export type Audio = z.infer<typeof Audio>;

File: src/youtube_automation/configuration/audio.py
"""オーディオ設定の責務別 dataclass（bobble 独自・optional）."""

from __future__ import annotations

from dataclasses import dataclass


@dataclass(frozen=True)
class Audio:
    """`audio` セクション（optional、bobble のみ）."""

    target_duration_min: float | None = None
    target_duration_max: float | None = None
    chapter_max: int = 100
